server/app/models/TrustedIntermediaryGroup.java

```diff
@@ -53,7 +55,8 @@ public String getDescription() {
     return this.description;
   }
 
-  public ImmutableList<Account> getManagedAccounts(Optional<String> search) {
+  public ImmutableList<Account> getManagedAccounts(
+      Optional<String> search, Optional<String> searchDate) {
```

Comment:
suggestion (blocking): document this method and state what happens if search and searchDate are present vs not